README.md

```diff
@@ -64,21 +63,21 @@ Using GitLink via the command line is very simple:
 
 When working with a content proxy or an alternative git VCS system that supports direct HTTP access to specific file revisions use the `-u` parameter with the custom raw content root URL
 
-    GitLink.exe c:\source\catel -u https://raw.githubusercontent.com/catel/catel
+    GitLink.exe -u https://raw.githubusercontent.com/catel/catel <pdbfile>
```

Comment:
Either way works. I moved it to the end because the USAGE doc printed by the cmdline library we're using follows the *nix pattern (for xplat friendly reasons) of defining switches first, then arguments later. So <pdbfile> being the only argument is printed last. But either order works, so if in our own docs we want to keep the pdb file listed as the first parameter, I can revert back to that here.